### ISSUE TITLE
Unusual error on step 2: 'The owner of Jackett directory is 'root'

### ISSUE THREAD

**Issue by catraba** [Jackett/Jackett#9005]
### Environment

**OS**: Ubuntu 20.04

**Jackett Version**: v0.16.728

**Are you using a proxy or VPN?** [yes/no] no

### Description

I try to do the second step but It's impossible. When I run sudo ./install_service_systemd.sh I get the message that I attached below.

### Logged Error Messages

ERROR: The owner of Jackett directory is 'root'.
Please, change the owner with the command 'chown <user>:<user> -R "/home/Jackett"'
The user <user> will be used to run Jackett.

**Comment by garfield69** [comment]
and have you run the command as described and then tried the second step again?
what happens?

**Comment by garfield69** [comment]
according to the script
https://github.com/Jackett/Jackett/blob/32ff4782f54f77dcb41f0a11a30eda1500713c5e/install_service_systemd.sh#L50-L58
it's expecting you to have a user name... how is it that you are running without a user name?
I'm going to leave this one for the linux team mmbers.

**Comment by jdkang** [comment]
It seems the (somewhat reasonable) expectation is the script is running as root but the service will be run as a dedicated user. 

Basically, the steps look something like this (keeping in mind `sudo`):

1. Create a system user
```
adduser \
    --system \
    --gecos "jackett service" \
    --disabled-password \
    --group \
    --home /var/lib/jackett \
    jackett
```
2. Extract and copy the release
```
cd `mktemp -d`
wget https://github.com/Jackett/Jackett/releases/download/v0.18.390/Jackett.Binaries.LinuxAMDx64.tar.gz
tar zxvf *.tar.gz
```

3. Copy binaries to service user's home and `chown`
```
cp -r ./Jackett /var/lib/jackett/
chown -r jackett:jackett /var/lib/jackett/Jackett
```

4. Run the script
```
cd /var/lib/jackett/Jackett
./install_service_systemd.sh
```

**Comment by FireOrion** [comment]
very good！

**Comment by cantcurecancer** [comment]
> systemctl status jackett
Thanks, but make sure it's "chown -R". Capital 'R' matters.

The Jackett systemd service install is by FAR the poorest implemented child of the servarr series and its install instructions gives the user no warning of some of dumb username choices it makes for you, as seen above. In reality, a blurb on the site saying "hey, I'm going to use the jellyfin username as the default, change this to: <input_user_here>" would be fine. Or at least take the less dumb route like radarr or sonarr and just create their own user and have it start the systemd service.

TL;DR if you have a problem with getting to the Jackett page, but it seems to have installed properly, you probably just need to change the owner of the main jackett install folder to something else then restart the service. You can create a jackett user, or you can reuse the radarr username, but I couldn't use the user 'jellyfin'.

**Comment by ilike2burnthing** [comment]
Jackett isn't part of the Servarr group.

I'm not sure if you were just using it as an example, but to be clear `jellyfin` does not exist anywhere in our code.

PRs are always welcome; if you think there are improvements to be made to the install instructions, please make them.
